### PR TITLE
build(deps): bump dart_code_metrics from 5.5.0 to 5.7.2 in /mews_pedantic

### PR DESCRIPTION
**Pull request by rhbrunetto**
#### Summary

- Upgrades `dart_code_metrics` to `5.7.2`

#### Testing steps

None.

#### Follow-up issues

None.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: mews_pedantic/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.14.0
+
+> Note: This release has breaking changes.
+
+- **BREAKING** **FEAT**: Upgrage dart_code_metrics to 5.7.2.
```
Comment (witwash) [inline mews_pedantic/CHANGELOG.md:5]: typo (should be **Upgrade**)

**Comment by witwash** [comment]
@rhbrunetto, thank you. I would accept it, but I can't publish the new version anyway. We have to wait for @ookami-kb.

**Comment by ookami-kb** [comment]
@rhbrunetto please also push the related tag.

**Comment by ookami-kb** [comment]
Updated in #350

**Pull request closed without merging**